spinnman/messages/scp/impl/scp_utils.py

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
So after discussion of exploding out this compution. Can you give me a direct mapping between 0,1,2 and word, 1/2 word and byte reading please.

thank you